src/library/scala/collection/Seq.scala

```diff
@@ -156,7 +156,7 @@ trait SeqOps[+A, +CC[_], +C] extends Any
     *  @return       a new $coll which contains all elements of `prefix` followed
     *                  by all the elements of this $coll.
     */
-  def prependedAll[B >: A](prefix: Iterable[B]): CC[B] = fromIterable(new View.Concat(prefix, this))
+  def prependedAll[B >: A](prefix: Iterable[B]): CC[B] = iterableFactory.from(new View.Concat(prefix, this))
```

Comment:
This could also take an `IterableOnce`. (And this was the case [in 2.12](https://www.scala-lang.org/api/current/scala/collection/Seq.html#++:[B](that:scala.collection.TraversableOnce[B]):Seq[B]))